src/Core/Banshee.WebBrowser/Banshee.WebBrowser.dll.config

```diff
@@ -0,0 +1,5 @@
+<configuration>
+  <dllmap dll="libgdk-3-0.dll" target="libgdk-3.dylib" os="osx"/>
+  <dllmap dll="libgdk-3-0.dll" target="libgdk-3.so.0" os="!windows,osx"/>
+  <dllmap dll="libgdk-3-0.dll" target="libgdk-3-0.dll" os="windows"/>
```

Comment:
nice approach to multi-platform :) there's no `linux` os value? (see the `mtp.m4` and the `config.in` file in the Mtp library for an alternative, I'm not sure which is best, this is certainly simpler)

Reply:
Interesting. I directly took it from Banshee.GStreamerSharp.dll.config.
Not sure what the right way is...
It does surely work on linux :wink:

Reply:
Sweet, if there's precedent for this way then let's do it this way - I had only seen the Mtp version - I'll update the Gtk.Extra library `feature/lite` to match